Sources/PubNub/APIs/Objects+PubNub.swift

```diff
@@ -474,9 +731,11 @@ public extension PubNub {
       configuration: requestConfig.customConfiguration ?? configuration
     )
 
-    route(router,
-          responseDecoder: GenericServiceResponseDecoder(),
-          custom: requestConfig) { result in
+    route(
```

Comment:
If we have explicit method names to set channel / user metadata, do we want to have the same for `remove`?

Reply:
1. Regarding removing User metadata, here's the method we have so far - which I deprecated in this PR:

```
@available(*, deprecated, renamed: "removeUserMetadata(_:custom:completion:)")
func remove(
   uuid metadataId: String?,
   custom requestConfig: RequestConfiguration = RequestConfiguration(),
   completion: ((Result<String, Error>) -> Void)?
)
```
From now, the recommended method for removing User metadata would be:

```
func removeUserMetadata(
   _ metadataId: String?,
   custom requestConfig: RequestConfiguration = RequestConfiguration(),
   completion: ((Result<String, Error>) -> Void)?
)
```

2. Regarding removing Channel metadata, nothing has changed in the scope of this PR so the customers will still use the same method as previously:

```
func remove(
   channel metadataId: String,
   custom requestConfig: RequestConfiguration = RequestConfiguration(),
   completion: ((Result<String, Error>) -> Void)?
)
```

Do you think is it OK? Or would you like to introduce the new method for removing Channel metadata and slightly change its name?

```
func removeChannelMetadata(
   _ metadataId: String,
   custom requestConfig: RequestConfiguration = RequestConfiguration(),
   completion: ((Result<String, Error>) -> Void)?
)
```

Reply:
It is just kind of weird that one metadata type `remove` strongly `emphasized` and for another we would know only if review argument label names.

Reply:
Am I right? Would you prefer this method? If so, I could introduce it:

```
func removeChannelMetadata(
   _ metadataId: String,
   custom requestConfig: RequestConfiguration = RequestConfiguration(),
   completion: ((Result<String, Error>) -> Void)?
)
```